classes/class-sailthru-horizon.php

```diff
@@ -197,6 +197,13 @@ public function register_plugin_scripts() {
 						wp_register_script( 'tag', plugin_dir_url( __DIR__ ) . 'js/tag.js', array('jquery') );
 						wp_localize_script( 'tag', 'tag', $params );
 						wp_enqueue_script( 'tag' , '', [], false, $this->js_in_footer());
+						add_filter( 'script_loader_tag', 'spm_async', 10, 3 ); // force SPM to load async
+						function spm_async( $tag, $handle, $src ) {
```

Comment:
from VIP team: place function at class-level

Reply:
I've tried this and it doesn't work.